### ISSUE TITLE
General Issue

### ISSUE THREAD

**Issue by BearHeathen** [kode80/VSCUnity#1]
I can't seem to get this plugin working. I don't get any compilation errors and a reboot didn't resolve. I am running Unity 5.0.1f1 Personal Edition on Windows 8.1. 

I placed the "VSCUnity" folder into Assets--->Editor as instructed. My editor does not change at all (or if it has I don't see where). Ergo, since I don't see the options I can't leverage the awesomeness of VS Code. I'm sure it's probably me doing something wrong. I'm relatively new to Unity so I apologize if that's the case.

Below are pictures of my hierarchy (to show where I put the file) and my toolbar (to show nothing is there). There is also nothing in the Editor View. 

![hierarchy](https://cloud.githubusercontent.com/assets/7244519/7447685/edba6838-f1ca-11e4-8ba4-4a49dfd87c2c.PNG)

![tollbar](https://cloud.githubusercontent.com/assets/7244519/7447687/f198bdf6-f1ca-11e4-86a6-33bb75483949.PNG)

**Comment by prime31** [comment]
Hey man, did you take a few minutes to look at the 2 source code files? Windows is not supported and you will see the a popup message dialog when attempting to use the menu items:

Adding Unity symbols to Visual Studio Code is not currently supported on " + Application.platform

kode80 and any other folks releasing open source code spend a lot of time producing stuff for free. Please do them a favor and take the time to explore what they are providing in detail on your own time before wasting their time with questions you can answer yourself with a small amount of due diligence.

**Comment by kode80** [comment]
At the moment it's OSX only, however fixing this for other platforms is just a case of grabbing the appropriate app path for that platform. If you look at the code it should be pretty self-explanatory what needs to be implemented to get this working on win/linux. I will do this myself at some point but OSX is my main dev platform (and windows users still have full VS) so it isn't high priority - if someone wants to take the reins I'll happily accept pull requests. ;)

**Comment by BearHeathen** [comment]
Thank you, kode80. I will take a gander and see if I can help out a bit. And prime31, I saw no dialog box whatsoever, and there were no menu items that I could even begin to attempt to use. The reason for this is fairly obvious now, of course. 

And I am aware of the necessity of due diligence. However, I failed to mention that I'm on a bit of a time crunch due to a project being due on the 8th. I didn't even open the code, to be perfectly honest. VS Code is an elegant solution that fits nicely in my web dev workflow and I was hoping there was room for it in my game dev workflow. 

It'd be nice to see this working, so I'll pull it and peruse it, but I don't expect to be able to really get cracking until 08 May. Thanks for this epic tool nonetheless.
